src/main/java/io/sprucehill/zalando/api/nativecart/model/Delivery.java

```diff
@@ -0,0 +1,34 @@
+package io.sprucehill.zalando.api.nativecart.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.sprucehill.zalando.api.model.Base;
+
+/**
+ * 
+ * @author dipteewarudkar
+ *
+ */
+public class Delivery extends Base {
+	
+	@JsonProperty
+	private String earliest;
```

Comment:
We should maybe use `Date` here instead of `String` and apply correct `@JsonFormat`